docs/entity_registry_index.md

```diff
@@ -9,14 +9,28 @@ Being registered has the advantage that the same entity will always get the same
 
 A user is also able to override the name of an entity in the entity registry. When set, the name of the entity registry is used in favor of the name the device might give itself.
 
-## Unique ID requirements
+## Unique ID
+
+It is important that it is not possible for the user to change the unique ID, because they system would lose all its settings related to it.
```

Comment:
```suggestion
It is important that it is not possible for the user to change the unique ID, because the system would lose all its settings related to it.
```